README.md

```diff
@@ -1,7 +1,39 @@
 # gainbridge-qa
 
+## Installation
+
+1. Clone the repository:
+
+   ```bash
+
+   git clone "repo url"
```

Comment:
> we have to keep git repository url as well. so that they can copy past and run it 
Sir, I have updated "repo url" text with git repository link

![1 gainBridge](https://github.com/SureshQAExpert/gainbridge-qa/assets/168629072/086f205c-3f6e-4697-ac91-83de83d00d66)